edb/schema/name.py

```diff
@@ -79,7 +79,7 @@ def is_qualified(name: Union[SchemaName, str]) -> bool:
 
 class UnqualifiedName(SchemaName):
 
-    def __new__(cls: Type[NameT], name: str) -> NameT:
+    def __new__(cls: Type[NameT], name: str) -> Any:
```

Comment:
You can omit the annotation on `cls` and change the return annotation to `UnqualifiedName`.

Reply:
This presents the same problem described in the comment above. (type: ignore -> Any; returning Any from function declared to return ...)